backends/chromeos/chromeos-utils.js

```diff
@@ -203,8 +203,8 @@ async function executeRemoteCommand(log, ssh, argv) {
 
   const output = await ssh.exec(executable, args, options);
 
-  // output.code == 0 means success.
-  if (output.code != 0) {
+  // output.code == 0 or output.code == null means success.
+  if (output.code != 0 && output.code != null) {
```

Comment:
nit; Inverting the condition to `!(output.code == 0 || output.code == null)` might be a bit easier to read since it matches the comment.

Reply:
Done